packages/sdk/src/types.ts

```diff
@@ -60,41 +60,63 @@ export type MakeBuyOrderParams = MakeSellOrderParams;
 export enum MarketplaceName {
   Opensea = "opensea",
   Trader = "trader",
-  Looksrare = "looksrare",
+  LooksRare = "looksrare",
 }
 
-interface Opensea {
+interface ErrorObj {
+  message?: string;
+}
+
+interface ErrorBase {
+  error?: ErrorObj;
+}
+
+interface OpenseaBase extends ErrorBase {
   marketplaceName: MarketplaceName.Opensea;
 }
 
-interface Trader {
+interface TraderBase extends ErrorBase {
   marketplaceName: MarketplaceName.Trader;
 }
 
-interface Looksrare {
-  marketplaceName: MarketplaceName.Looksrare;
+interface LooksRareBase extends ErrorBase {
+  marketplaceName: MarketplaceName.LooksRare;
 }
 
-export interface OpenseaOrder extends Opensea {
-  marketplaceOrder: _OpenseaOrder;
+export interface NormalizedAsset {
+  contractAddress: string;
+  tokenId?: string;
+  type?: string;
+  amount: string;
```

Comment:
Thowing it out for disucssion - since we're taking in Asset amount in `bigint`, IMO we should export amount in `NormalizedAsset` as `bigint` as well for consistency

Lemme know what you guys think @kainanaina @leeyikjiun

Reply:
I think string, BigNumber, or bigint is fine, just be consistent. I used bigint over BigNumber to reduce a dependency, and not string to prevent users from passing random strings that is not a number. Let the typing do its job rather than defensive guard all over the place.